libs/langchain/langchain/vectorstores/redis/base.py

```diff
@@ -707,9 +707,14 @@ def add_texts(
 
         # Write data to redis
         pipeline = self.client.pipeline(transaction=False)
+        if not embeddings:
+            embeddings = []
+            for i, text in enumerate(texts):
```

Comment:
The embeddings are made a few lines down? if we did create the index here, why loop through them all?